config/initializers/doorkeeper.rb

```diff
@@ -100,3 +100,4 @@
   # set to true if you want this to be allowed
   # wildcard_redirect_uri false
 end
+Doorkeeper.configuration.token_grant_types << "password"
```

Comment:
That comment only means that it used to be on by default, but needs to be added explicitly as of Doorkeeper 2.1. I've checked the code, and adding at line 86 will definitely work :)